broker/value_added/xmatch.py

```diff
@@ -10,25 +10,77 @@
 """
```

Comment:
For this module, I suggest thinking through when you want to `print`, `warn` or `log`.